test/integration/helpers/PathToWindowsNode

```diff
@@ -1 +1 @@
-C:/Program Files (x86)/Appium/node_modules/appium/bin/appium.js
\ No newline at end of file
+%USERPROFILE%/AppData/Roaming/npm/node_modules/appium/lib/appium.js
```

Comment:
I believe this is not necessary and the Appium path could be detected dynamically based on the actual npm output:

<img width="883" alt="win_path" src="https://github.com/appium/dotnet-client/assets/7767781/5839100a-02e2-4e77-83df-4d0a9d71eb71">

Reply:
Hi @mykola-mokhnach, Not sure excalty what you mean by that, but we basically just use this path to test that we can pass a specific appium node path to the server.

Reply:
I mean it is not necessary to hardcode the path to the appium script (which by the way should point to index.js in the module root rather than lib/appium.js)

Also the npm prefix itself should be retrieved dynamically from the npm config rather than hardcoded

Reply:
OK, I'll see what I can do about that.

Reply:
@mykola-mokhnach, same go to the Mac and Linux path?
we currently use:
/Applications/Appium.app/Contents/Resources/node_modules/appium/bin/appium.js

Reply:
right, the algorithm should be cross-platform
I assume the only difference is that in *nix systems `npm` is an executable while on Windows it is a .cmd script

Reply:
https://github.com/appium/java-client/blob/f875c3b86aa0f14e27a91d0a3d1f8e51d02e9054/src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java#L147

Reply:
@mykola-mokhnach, I've pushed a big refactor, let me know if it's better now.
I will need to remove the other node's paths if we are good.
I also still need to test this on Unix/Mac system

Reply:
@mykola-mokhnach, while checking on MAC I encountered an issue with the path
index.js location on the Mac:
`/usr/local/lib/node_modules/appium/index.js`
![image](https://github.com/appium/dotnet-client/assets/59066376/c79c8fae-d2b2-4002-949a-e9782526b1b7)

but when running `which npm`, I get this npm prefix:
`/usr/local/bin/npm`
![image](https://github.com/appium/dotnet-client/assets/59066376/79e69e52-9eba-437d-adec-3724987205e5)

any suggestions on how to overcome this?

Reply:
Looks like you are mixing npm install prefix and npm install path. The first one is where npm installs global modules and the second is just the path to the package manager binary itself

Reply:
So I should use a different command other than 'which npm' ?

Reply:
Why not? Depending on what you want to achieve. If this is to find the npm script path then why not?

Reply:
so `npm list -g --depth=0` gives me the correct path, I just need to modify the `PathToNode` so it will work with Unix as well.
I will also need to see how it works with Windows when using the `npm list -g --depth=0`

Reply:
usually the above command takes more time because it needs to scan multiple packages. Why not to use `npm config get prefix` or `npm -g root` ?

Reply:
Moved to `npm -g root`